runtime/src/bank/partitioned_epoch_rewards/calculation.rs

```diff
@@ -370,37 +367,25 @@ impl Bank {
                     });
 
                     let stake_pubkey = **stake_pubkey;
-                    let stake_account = (*stake_account).to_owned();
-
                     let vote_pubkey = stake_account.delegation().voter_pubkey;
-                    let (mut stake_account, stake_state) =
-                        <(AccountSharedData, StakeStateV2)>::from(stake_account);
                     let vote_account = get_vote_account(&vote_pubkey)?;
                     if vote_account.owner() != &solana_vote_program {
                         return None;
                     }
                     let vote_state = vote_account.vote_state();
-
-                    let pre_lamport = stake_account.lamports();
+                    let mut stake_state = *stake_account.stake_state();
 
                     let redeemed = redeem_rewards(
                         rewarded_epoch,
-                        stake_state,
-                        &mut stake_account,
+                        &mut stake_state,
                         vote_state,
                         &point_value,
                         stake_history,
                         reward_calc_tracer.as_ref(),
                         new_warmup_cooldown_rate_epoch,
                     );
 
-                    let post_lamport = stake_account.lamports();
-
                     if let Ok((stakers_reward, voters_reward)) = redeemed {
-                        debug!(
-                            "calculated reward: {} {} {} {}",
-                            stake_pubkey, pre_lamport, post_lamport, stakers_reward
-                        );
```

Comment:
I'm ambivalent, but seems like it could be useful to recreate this log in the distribution flow. Thoughts?

Reply:
I think if we wanted to debug the distribution flow we would add logs to a custom ledger tool anyways. So I don't think it's that important to be able to turn on logs for this on a running validator.